src/main/resources/api.yaml

```diff
@@ -759,6 +759,58 @@ paths:
           description: User do not have permissions to delete permission to the resource
         '404':
           description: Resource not found
+  /comments/resources/{resourceType}/{resourceId}:
```

Comment:
As talked earlier, I suggest changing convention to `/resources/{resourceType}/{resourceId}/comments:`